entries/s/sendcloud.com.json

```
@@ -0,0 +1,24 @@
+{
+  "Example Site": {
```

This element specifies the name of the service. As such it should have the value `Sendcloud`
```suggestion
  "Sendcloud": {
```